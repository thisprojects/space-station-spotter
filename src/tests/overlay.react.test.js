import React from "react";
import OverLay, {
  Errors,
  GeoCode,
  InputAndButton,
} from "../components/overlay-components";
import renderer from "react-test-renderer";

test("Errors Matches Snapshot", () => {
  const component = renderer.create(<Errors />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Overlay Matches Snapshot", () => {
  const component = renderer.create(<OverLay />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("GeoCode Matches Snapshot", () => {
  const component = renderer.create(<GeoCode />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Location Input and Button Matches Snapshot", () => {
  const component = renderer.create(<InputAndButton />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


