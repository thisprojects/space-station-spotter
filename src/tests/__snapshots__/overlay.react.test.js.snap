// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors Matches Snapshot 1`] = `null`;

exports[`GeoCode Matches Snapshot 1`] = `
<div
  className="geocode"
>
  <button
    onClick={[Function]}
  >
    Use Current Location
  </button>
</div>
`;

exports[`Location Input and Button Matches Snapshot 1`] = `
Array [
  <div
    className="location-input-wrapper"
  >
    <input
      id="location-input"
      placeholder="Choose Location"
      type="text"
    />
  </div>,
  <div
    className="submit-button"
  >
    <button
      onClick={[Function]}
    >
      Submit
    </button>
  </div>,
]
`;

exports[`Overlay Matches Snapshot 1`] = `null`;
